### DISCUSSION TITLE
Budibase Launchstack

### DISCUSSION THREAD

**Discussion by davidjeba** [Budibase/budibase#7436] [Ideas]
Recommending/Requesting to have a section for deployment ready full scale applications

budibase.com/launchpad - to list full scale applications ready to be launched and further customizations can be done

1.Real Estate portal - $999

- Login - Oauth system
- Property owner login
- Buyer login
- Property listing [faceted]
- Sponsord Ad system to boost listing 
- Buyer request/response system
- Auction type bidding system
- Traffic reports
- Notification system

2.Vehicle Listing Portal - $999

- Login - Oauth system
- brand owner login
- Buyer/seller login
- vehicle listing [faceted]
- Sponsord Ad system to boost listing 
- Buyer request/response system
- Auction type bidding system
- Traffic reports
- Notification system

3.A Frontend and Backend for Swiggy like marketplace - $1999
4.A Frontend and Backend for neighbor .com like space rental system - $1999
5.A Frontend and Backend for Flippa.com marketplace - $1999

For developers who can afford to pay, let them pay and experiment completely built platform on budibase.

**Comment by andz-bb** [comment]
Hey @davidjeba - thanks for the idea! We already have [a lot of templates available for free here](https://budibase.com/templates/) that can easily be customised. We are focusing on making Budibase the easiest way to build great internal apps. Some of the features of the apps you are suggesting wouldn't be possible to build right now but our next release will give developers the opportunity to contribute their own components and integrations.

**Reply by davidjeba** [reply]
Templates are basic foundation for something to be built on
Starterkits are a little advanced version of templates
LaunchStack is fully ready to go complete system which may requie a max of 1000hrs for customizations 
budibase can easily outcompete Outsystems, builder.ai like platforms by offering more powerful systems to the developer network

**Reply by joebudi** [reply]
Thank you for advice @davidjeba . This is certainly a direction we plan to go in, but first, we need to improve on the extensibility of Budibase - this is a major focus for Budibase 2.0 (coming very soon). 

Once this phase is complete, we hope to build a community marketplace, which will consist of components, integrations, and templates/starterkits.
